### ISSUE TITLE
Not able to run with docker-compose

### ISSUE THREAD

**Issue by alex2005git** [Countly/countly-server#1172]
### Expected Behavior

Executing `docker-compose up` and everything working as intended.

### Current Behavior

`countly-api` lots of cannot find module errors  
```bash
countly-api_1       | [docker] Installing onboarding":
countly-api_1       | module.js:550
countly-api_1       |     throw err;
countly-api_1       |     ^
countly-api_1       | 
countly-api_1       | Error: Cannot find module '/opt/countly/plugins/onboarding"/install.js'
countly-api_1       |     at Function.Module._resolveFilename (module.js:548:15)
countly-api_1       |     at Function.Module._load (module.js:475:25)
countly-api_1       |     at Function.Module.runMain (module.js:694:10)
countly-api_1       |     at startup (bootstrap_node.js:204:16)
countly-api_1       |     at bootstrap_node.js:625:3
countly-api_1       | [docker] Done installing onboarding".
countly-api_1       | 2020-02-28T15:55:37.736Z: INFO    [jobs:manager]  Starting job manager in 6
countly-api_1       | 2020-02-28T15:55:37.750Z: INFO    [jobs:scanner]  Checking plugins ["mobile","web","desktop","plugins","density","locale","browser","sources","views","enterpriseinfo","logger","systemlogs","errorlogs","populator","reports","crashes","push","star-rating","slipping-away-users","compare","server-stats","dbviewer","assistant","plugin-upload","times-of-day","compliance-hub","video-intelligence-monetization","alerts","onboarding\""]
countly-api_1       | Starting master
countly-api_1       | Error: Cannot find module './onboarding"/api/api'
countly-api_1       |     at Function.Module._resolveFilename (module.js:548:15)
countly-api_1       |     at Function.Module._load (module.js:475:25)
countly-api_1       |     at Module.require (module.js:597:17)
countly-api_1       |     at require (internal/module.js:11:18)
countly-api_1       |     at pluginManager.init (/opt/countly/plugins/pluginManager.js:66:43)
countly-api_1       |     at Object.<anonymous> (/opt/countly/api/api.js:151:9)
countly-api_1       |     at Module._compile (module.js:653:30)
countly-api_1       |     at Object.Module._extensions..js (module.js:664:10)
countly-api_1       |     at Module.load (module.js:566:32)
countly-api_1       |     at tryModuleLoad (module.js:506:12)
countly-api_1       |     at Function.Module._load (module.js:498:3)
countly-api_1       |     at Function.Module.runMain (module.js:694:10)
countly-api_1       |     at startup (bootstrap_node.js:204:16)
countly-api_1       |     at bootstrap_node.js:625:3
countly-api_1       | 2020-02-28T15:55:38.504Z: INFO    [jobs:ipc:6]    Attached to cluster in Central 6
```
...
```bash
countly-api_1       | Error: Cannot find module './onboarding"/api/api'
countly-api_1       |     at Function.Module._resolveFilename (module.js:548:15)
countly-api_1       |     at Function.Module._load (module.js:475:25)
countly-api_1       |     at Module.require (module.js:597:17)
countly-api_1       |     at require (internal/module.js:11:18)
countly-api_1       |     at pluginManager.init (/opt/countly/plugins/pluginManager.js:66:43)
countly-api_1       |     at Object.<anonymous> (/opt/countly/api/api.js:151:9)
countly-api_1       |     at Module._compile (module.js:653:30)
countly-api_1       |     at Object.Module._extensions..js (module.js:664:10)
countly-api_1       |     at Module.load (module.js:566:32)
countly-api_1       |     at tryModuleLoad (module.js:506:12)
countly-api_1       |     at Function.Module._load (module.js:498:3)
countly-api_1       |     at Function.Module.runMain (module.js:694:10)
countly-api_1       |     at startup (bootstrap_node.js:204:16)
countly-api_1       |     at bootstrap_node.js:625:3
countly-api_1       | Starting worker 233 parent: 6
countly-api_1       | Error: Cannot find module './onboarding"/api/api'
countly-api_1       |     at Function.Module._resolveFilename (module.js:548:15)
countly-api_1       |     at Function.Module._load (module.js:475:25)
countly-api_1       |     at Module.require (module.js:597:17)
countly-api_1       |     at require (internal/module.js:11:18)
countly-api_1       |     at pluginManager.init (/opt/countly/plugins/pluginManager.js:66:43)
countly-api_1       |     at Object.<anonymous> (/opt/countly/api/api.js:151:9)
countly-api_1       |     at Module._compile (module.js:653:30)
countly-api_1       |     at Object.Module._extensions..js (module.js:664:10)
countly-api_1       |     at Module.load (module.js:566:32)
countly-api_1       |     at tryModuleLoad (module.js:506:12)
countly-api_1       |     at Function.Module._load (module.js:498:3)
countly-api_1       |     at Function.Module.runMain (module.js:694:10)
countly-api_1       |     at startup (bootstrap_node.js:204:16)
countly-api_1       |     at bootstrap_node.js:625:3
countly-api_1       | Starting worker 263 parent: 6
```

`counly-frontend` a lot of the same module errors
```bash
countly-frontend_1  | [docker] Installing onboarding":
countly-frontend_1  | module.js:550
countly-frontend_1  |     throw err;
countly-frontend_1  |     ^
countly-frontend_1  | 
countly-frontend_1  | Error: Cannot find module '/opt/countly/plugins/onboarding"/install.js'
countly-frontend_1  |     at Function.Module._resolveFilename (module.js:548:15)
countly-frontend_1  |     at Function.Module._load (module.js:475:25)
countly-frontend_1  |     at Function.Module.runMain (module.js:694:10)
countly-frontend_1  |     at startup (bootstrap_node.js:204:16)
countly-frontend_1  |     at bootstrap_node.js:625:3
countly-frontend_1  | [docker] Done installing onboarding".
countly-frontend_1  | Running "concat:dom" (concat) task
```
...
```bash
countly-frontend_1  | Running "concat:locales_pl" (concat) task
countly-frontend_1  | 
countly-frontend_1  | Done.
countly-frontend_1  | Error: Cannot find module './onboarding"/frontend/app'
countly-frontend_1  |     at Function.Module._resolveFilename (module.js:548:15)
countly-frontend_1  |     at Function.Module._load (module.js:475:25)
countly-frontend_1  |     at Module.require (module.js:597:17)
countly-frontend_1  |     at require (internal/module.js:11:18)
countly-frontend_1  |     at pluginManager.loadAppStatic (/opt/countly/plugins/pluginManager.js:490:30)
countly-frontend_1  |     at Object.<anonymous> (/opt/countly/frontend/express/app.js:412:9)
countly-frontend_1  |     at Module._compile (module.js:653:30)
countly-frontend_1  |     at Object.Module._extensions..js (module.js:664:10)
countly-frontend_1  |     at Module.load (module.js:566:32)
countly-frontend_1  |     at tryModuleLoad (module.js:506:12)
countly-frontend_1  |     at Function.Module._load (module.js:498:3)
countly-frontend_1  |     at Function.Module.runMain (module.js:694:10)
countly-frontend_1  |     at startup (bootstrap_node.js:204:16)
countly-frontend_1  |     at bootstrap_node.js:625:3
```

### Steps to Reproduce (for bugs)

1. clone git repo
2. docker compose up -d mongodb
3. docker-compose up countly-api
4. docker-compose up countly-frontend

### More Description

Also shouldn't some ports be added to the docker-compose.yml file so that I can access them from localhost?

### Your Environment

* Operating System and version: Ubuntu 18.04 x64
* Any changes you have made to your instance:

**Comment by adolfdsilva** [comment]
+1

**Comment by stirante** [comment]
Problem is here https://github.com/Countly/countly-server/blob/master/docker-compose.yml#L25
and here https://github.com/Countly/countly-server/blob/master/docker-compose.yml#L50

Either remove double quote from the end or add it at the beginning.

**Comment by alexsegura** [comment]
I think I have the same problem more or less, I have this in Docker logs (notice there is **not** the double-quote issue):

```
Error: Cannot find module './errorlogs/frontend/app'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at pluginManager.loadAppStatic (/opt/countly/plugins/pluginManager.js:507:30)
    at Object.<anonymous> (/opt/countly/frontend/express/app.js:415:9)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Error: Cannot find module './plugin-upload/frontend/app'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at pluginManager.loadAppStatic (/opt/countly/plugins/pluginManager.js:507:30)
    at Object.<anonymous> (/opt/countly/frontend/express/app.js:415:9)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
```

I struggle to understand what happens. 
Disabling those 2 plugins in `COUNTLY_PLUGINS` env var makes the error disappear.

**Comment by iartem** [comment]
Forgot about that one, sorry. 

So long story short, some plugins don't make any sense in Docker, so we just remove them during image build. That's why those errors appear when you still have them in `COUNTLY_PLUGINS`. 

For some reason I've made that mistake with quotes and thought removing them would make the issue resolved, but the issue had a second thing - I though I removed those extra plugins from everywhere, but it seems I forgot to remove them from docker-compose file.
